newsroom/topics/views.py

```diff
@@ -54,6 +54,26 @@ def is_user_topic(topic_id, user_id):
     return False
 
 
+def get_topic_url(topic):
```

Comment:
wondering if we should rather use `topic=<topic-id>` and handle it on server, if topics are editable the url would have to change, but if it was shared already it won't work

Reply:
We could use id too. Afaik user cannot edit a topic (except name) but they can delete and recreate of course. Then shared email won't be valid. I was inclined to use URL as one feedback from AAP few weeks back was to use URL for the dropdown filters in agenda (which I did local store).

Reply:
I see, if the query is not editable it's ok